src/omnicore_pending.cpp

```diff
@@ -81,13 +87,13 @@ void PendingAdd(const uint256 txid, const std::string& sendingAddress, const std
  *
  * NOTE: this is currently called for every bitcoin transaction prior to running through the parser
  */
-void PendingDelete(const uint256 txid)
+void PendingDelete(const uint256& txid)
 {
     PendingMap::iterator it = my_pending.find(txid);
     if (it != my_pending.end()) {
         CMPPending *p_pending = &(it->second);
         int64_t src_amount = getMPbalance(p_pending->src, p_pending->prop, PENDING);
-        if (msc_debug_pending) file_log("%s(%s): amount=%ld\n", __FUNCTION__, txid.GetHex().c_str(), src_amount);
+        if (msc_debug_pending) file_log("%s(%s): amount=%d\n", __FUNCTION__, txid.GetHex(), src_amount);
```

Comment:
`LogPrintStr`, `PrintToConsole()`, `file_log()`, ..., use tinyformat, which is smarter, and works just fine with `%d` and `%f` for any number, and `%s` works without `c_str()`.

Maybe it's just personal taste, but in my opinion this provides greater freedom and less room for errors. Say for example there is an `uint64_t`, which is replaced by `int` at some point, and the specifier `%lu` is not updated, then the results are annoying compiler warnings.. :)